packages/expo-store-review/src/StoreReview.ts

```diff
@@ -41,10 +44,10 @@ export async function requestReview(): Promise<void> {
 }
 
 /**
- * Get your app's store URLs from `app.config.js` or `app.json`:
- * - iOS: https://docs.expo.io/versions/latest/workflow/configuration#appstoreurlurl-to-your-app-on-the-apple-app-store-if-you-have-deployed-it-there-this-is-used-to-link-to-your-store-page-from-your-expo-project-page-if-your-app-is-public
- * - Android: https://docs.expo.io/versions/latest/workflow/configuration#playstoreurlurl-to-your-app-on-the-google-play-store-if-you-have-deployed-it-there-this-is-used-to-link-to-your-store-page-from-your-expo-project-page-if-your-app-is-public
- * - web: returns `null`
+ * This uses the `Constants` API to get the `Constants.manifest.ios.appStoreUrl` on iOS, or the
+ * `Constants.manifest.android.playStoreUrl` on Android.
+ *
+ * On Web and in the bare workflow, this will return `null`.
```

Comment:
```suggestion
 * On Web this will return `null`.
```

we embed the manifest in bare workflow apps now